index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Барбершоп «Бородинский»</title>
+</head>
+
+<body>
+    <header class="main-header">
+        <nav class="main-navigation">
+            <a class="main-header-logo">
+                <img src="img/index-logo.png" width="368" height="153" alt="Логотип барбершопа «Бородинский»">
+            </a>
+            <ul class="site-navigation">
+                <a href="info.html">Информация</a></li>
+                <li><a href="news.html">Новости</a></li>
```

Comment:
```suggestion
                <li>
                    <a href="news.html">Новости</a>
                </li>
```

Ещё вот по кодстайлу поправить.